WORKSPACE

```diff
@@ -18,76 +18,28 @@
 
 workspace(name = "grakn_core")
 
-load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-
-
-########################################
-#        Remote Build Execution        #
-########################################
-
-http_archive(
-  name = "bazel_toolchains",
-  sha256 = "07a81ee03f5feae354c9f98c884e8e886914856fb2b6a63cba4619ef10aaaf0b",
-  strip_prefix = "bazel-toolchains-31b5dc8c4e9c7fd3f5f4d04c6714f2ce87b126c1",
-  urls = [
-    "https://mirror.bazel.build/github.com/bazelbuild/bazel-toolchains/archive/31b5dc8c4e9c7fd3f5f4d04c6714f2ce87b126c1.tar.gz",
-    "https://github.com/bazelbuild/bazel-toolchains/archive/31b5dc8c4e9c7fd3f5f4d04c6714f2ce87b126c1.tar.gz",
-  ],
-)
-
 
 ####################
 # Load Build Tools #
 ####################
 
 # Load additional build tools, such bazel-deps and unused-deps
+load("//dependencies/tools:bazel/dependencies.bzl", "tools_bazel_dependencies")
+tools_bazel_dependencies()
+
 load("//dependencies/tools:dependencies.bzl", "tools_dependencies")
```

Comment:
Why are we splitting the two below @lolski :
- `//dependencies/tools:dependencies.bzl`
- `//dependencies/tools:bazel/dependencies.bzl`
Both seem to contain tools from Bazel. They "feel" like they should belong together in one macro. What do you think?

Reply:
@haikalpribadi the reason I split it was because I was following this instruction in https://github.com/graknlabs/grakn/issues/4870:
```
Move `http_archive(name="bazel_toolchains")` to `//dependencies/tools/dependencies.bzl`, then move `//dependencies/tools/dependencies.bzl` to `//dependencies/tools/bazel/dependencies.bzl`
```

shall we combine both of them as one file?

Reply:
ah yes, sorry I should have been clearer. That point implied that we should merge them @lolski ;)

Reply:
@haikalpribadi Cool! In the last commit, `//dependencies/tools:bazel/dependencies.bzl` has been merged to  `//dependencies/tools:dependencies.bzl`